### PR TITLE
feat: Adds ChannelCount as a filter to the Player Select Audio Track Method

### PR DESCRIPTION
**Pull request by jlnabais**
Adds channelCount to the Select Audio Track method, this allows for selecting a audio track based on the channel count when the language, codec and role are the same.

Issue: https://github.com/shaka-project/shaka-player/issues/4550

### CODE REVIEW

**Comment by avelad** [comment]
Please, sign the CLA. Thanks!

**Comment by jlnabais** [comment]
> Please, sign the CLA. Thanks!

Done :)

File: lib/player.js [hunk 1]
```diff
@@ -4047,15 +4047,16 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
    *
    * @param {string} language
    * @param {string=} role
+   * @param {number=} channelsCount
```
Comment (theodab) [inline lib/player.js:4050]: You should also update the JSDoc above.

**Review by theodab — CHANGES_REQUESTED** [review]
Looks good to me otherwise.

File: lib/player.js (hunk 1 above)
Comment (jlnabais) [reply lib/player.js:4050]: Done.

**Review by theodab — APPROVED** [review]

**Pull request merged**